packages/office-addin-debugging/src/start.ts

```diff
@@ -267,6 +286,10 @@ export async function waitUntil(callback: (() => Promise<boolean>), retryCount:
     return done;
 }
 
+export async function waitUntilDevBuildIsComplete(sourceLocation: string, retryCount: number = 30, retryDelay: number = 1000): Promise<boolean> {
```

Comment:
Along lines of previous comments, change name to `waitUntilUrlIsAvailalbe` and change `sourceLocation` to `url`.